### ISSUE TITLE
exports.default ?

### ISSUE THREAD

**Issue by stevenvachon** [jcollado/promisify-function#97]
``` js
exports.default = promisify;
```

so, we cannot use this as the documentation describes. Instead, we need to access its property:

``` js
const promisify = require("promisify-function").default;
```

Sloppy.

**Comment by jcollado** [comment]
@stevenvachon Thanks for reporting the inconsistency in the documentation.

The change in the export happened when moving the code to es2015. I believe the `default` export is correct as it is now, but the documentation should say something like:
`import promisify from 'promisify-function'`

**Comment by stevenvachon** [comment]
Yes, but that would require us to transpile our modules. In my case, this would be the only package to justify such.

**Comment by adbutterfield** [comment]
I came across the same issue today. I was confused at first too, until I did a bit of research and came across this:
https://github.com/babel/babel/issues/2212
